velox/functions/prestosql/window/tests/WindowTestBase.h

```diff
@@ -57,12 +49,14 @@ class WindowTestBase : public exec::test::OperatorTestBase {
   // including input parameters and open/close braces. e.g. rank(), ntile(5)
   void testTwoColumnOverClauses(
       const std::vector<RowVectorPtr>& input,
-      const std::string& windowFunction);
+      const std::string& windowFunction,
+      const std::string& overClause,
```

Comment:
The reason for this function was to generalize the overclauses across the tests. Should we separate the frame generation than overload this method ?

Reply:
Thanks for the suggestion, I parameterized on <overClause, frameClause> tuple since it would be easier to debug test failures by breaking down into separate tests. I will revert the parameterization on overClause though if that would be better, please let me know!